sdcm/nemesis.py

```diff
@@ -97,11 +97,11 @@ class NoMandatoryParameter(Exception):
     """ raised from within a nemesis execution to skip this nemesis"""
 
 
-class DefaultValue:
+class DefaultValue:  # pylint: disable=too-few-public-methods
     """
     This is class is intended to be used as default value for the cases when None is not applicable
     """
-    pass
+    ...
```

Comment:
Having docstring allows you have nothing in the body

Reply:
Yep